### PR TITLE
BankID OIDC flow 

### PR DESCRIPTION
**Pull request by k-irishina**
Added ~~authorizationURL~~ authenticationURL tokenId to the notification towards merchant we'll send after registering permissions for enrolment.

### CODE REVIEW

File: openapi/integrator/wallet/server.yaml [hunk 1]
```diff
@@ -91,6 +91,19 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /payment-tokens/authorization-response:
+    get:
+      summary: The redirection endpoint the authorization server directs to.
```
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:96]: This does not necessarily need to be used only for _HTTP redirect binding_ only, so we could rephrase to something like this:

> Endpoint used by the authentication provider to provide the authentication response.

File: openapi/integrator/wallet/server.yaml [hunk 2]
```diff
@@ -142,3 +155,29 @@ components:
           $ref: '../components.yaml#/components/schemas/merchantDisplayName'
         merchantReference:
           $ref: '../components.yaml#/components/schemas/merchantReference'
+
+  parameters:
+    code:
```
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:160]: oauth2AuthorisationCode

File: openapi/integrator/wallet/server.yaml [hunk 3]
```diff
@@ -142,3 +155,29 @@ components:
...
+      name: code
+      in: query
+      required: false
+      schema:
+        type: string
+    state:
```
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:166]: oauth2State

File: openapi/integrator/wallet/server.yaml [hunk 4]
```diff
@@ -142,3 +155,29 @@ components:
...
+      name: state
+      in: query
+      required: true
+      schema:
+        type: string
+    error:
```
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:172]: oauth2Error

File: openapi/integrator/wallet/server.yaml [hunk 5]
```diff
@@ -142,3 +155,29 @@ components:
...
+      name: error
+      in: query
+      required: false
+      schema:
+        type: string
+    error_description:
```
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:178]: oauth2ErrorDescription

**Review by thxmasj — CHANGES_REQUESTED** [review]

File: openapi/integrator/wallet/client.yaml [hunk 6]
```diff
@@ -254,10 +254,13 @@ components:
...
       type: object
       required:
-        - bindingMessage
+        - authorizationURL
+        - tokenId
       properties:
-        bindingMessage:
-          $ref: '../components.yaml#/components/schemas/authenticationBindingMessage'
+        authorizationURL:
+          $ref: '../components.yaml#/components/schemas/authorizationURL'
```
[the remark this answers is not in the document]
Comment (k-irishina) [reply openapi/integrator/wallet/client.yaml:261]: Then we're not aligned with oauth2 spec, but ok. :-)

**Review by thxmasj — APPROVED** [review]
Remember to not merge until just before we merge the platform.

**Pull request merged**